### ISSUE TITLE
Learning GraphQL

### ISSUE THREAD

**Issue by eveporcello** [eveporcello/oneblog#3]
![](https://github.com/eveporcello/oneblog/blob/master/img/704/1*LhZpvVSzr1-1b5AXRMfnyw.png?raw=true)

# Learning GraphQL

## Announcing new resources to help you build apps with Apollo

I’ve never been surfing. To me, surfing forces you to put a lot of trust in the ocean, and I do not trust the ocean at all. If I ever felt the need to go surfing, I probably wouldn’t head out to Mavericks on Day One. Instead, I’d start in a lesson, doing that thing where you lie on your surfboard on the beach and pretend to surf until the instructor trusts you not to drown.

With our company [Moon Highway](https://www.moonhighway.com), [Alex Banks](https://www.twitter.com/moontahoe) and I have spent countless hours researching, working with, and writing about GraphQL to ensure that our students and readers do not drown. We know that GraphQL is the future of APIs, but we also know that people don’t adopt new technologies if their first introduction is confusing, complicated, and not fun. If people have a great experience learning, they’ll get excited about working with GraphQL.

Getting people excited to build awesome products with GraphQL is our main goal, and now we have a book for that!

![](https://github.com/eveporcello/oneblog/blob/master/img/704/1*I7GmpIB0L08C8LXzsYIKtw.jpeg?raw=true "O’Reilly’s Learning GraphQL")

# **_Announcing Learning GraphQL_**

We’re delighted to announce that our second O’Reilly book, [_Learning GraphQL_](http://shop.oreilly.com/product/0636920137269.do), started shipping this week! Starting with an overview of GraphQL history, the book begins with a look at core GraphQL topics like schema design and the GraphQL query language. From there, we build a fullstack application that incorporates Apollo Server 2.0, Apollo Client, and React. We also cover advanced topics like database integration, GitHub authorization, and managing your API in production with Apollo Engine.

The Apollo team has been hugely supportive of this book throughout the entire process. They’ve provided us with thorough tech reviews and shared information about their technology roadmap, which means this book has the freshest content available. This collaboration with the folks from Apollo has been phenomenal, and this leads me to a couple other exciting announcements:

![](https://github.com/eveporcello/oneblog/blob/master/img/704/1*07FE7SG4qy1bahcweb3Usg.png?raw=true)

# **Apollo Training Partner Program**

We have signed on to be Apollo’s first Official Training Partner! Being part of this new partner program will allow us to focus on what we’re passionate about: delivering comprehensive, hands-on GraphQL training in a fun and encouraging environment for developers and companies around the world. This close partnership with the Apollo team will also ensure that our courses remain relevant and up-to-date.

If your team needs training on how to build production-ready apps with Apollo, please reach out to [team@apollographql.com](mailto:team@apollographql.com) to learn more.

![](https://github.com/eveporcello/oneblog/blob/master/img/704/1*erEy0kAge67XcDwhgUr-Xw.jpeg?raw=true)

# **Fullstack Bootcamp at GraphQL Summit**

On November 7–8, Apollo will host [GraphQL Summit](http://summit.graphql.com) in San Francisco, the largest GraphQL developer event in the world. To kick off the conference week, we’ll be teaching one-day [fullstack GraphQL Bootcamps](https://summit.graphql.com/#bootcamp) on Monday Nov 5 and Tuesday Nov 6 before the main conference starts. Join us for this fast-paced, hands-on course where you’ll learn to build powerful GraphQL apps with the Apollo platform. Of course, you’ll get a signed copy of new [_Learning GraphQL_](http://shop.oreilly.com/product/0636920137269.do) book as well!

I’ll also be giving a talk at GraphQL Summit that will focus on our experiences teaching GraphQL and how you can help others in your organizations learn about GraphQL and the Apollo platform.

We are thrilled to be working with Apollo to bring more engineers into the community with effective, hands-on training. Surf’s up! The tide is rising for GraphQL!

[https://upscri.be/64a355](https://upscri.be/64a355 "Join us on Apollo Slack! ")

```backmatter
{"source":"medium","postId":"4dbb7d8e1893","publishedDate":1536261249789,"url":"https://blog.apollographql.com/learning-graphql-4dbb7d8e1893"}
```

**Comment by sgrove** [comment]
Are there any future events where I can get a signed copy or two of [Learning GraphQL](http://shop.oreilly.com/product/0636920137269.do)? 🤔💯

**Comment by eveporcello** [comment]
Not sure if this is a real request or a fun demo of these features but I can always send books your way, Sean! :)

**Comment by sgrove** [comment]
Oh, always real - giving away *signed* copies would be even cooler! https://youtu.be/JilN_PvQOqs?t=4427

**Comment by eveporcello** [comment]
yes!! 100%! I can send you a bunch. Will follow up to get a good location to mail some signed copies to!
